application/views/template_view.php

```diff
@@ -9,21 +9,95 @@
 	<script src="https://code.jquery.com/jquery-3.7.1.min.js" integrity="sha256-/JqT3SQfawRcv/BIHPThkBvs0OEvtFFmqPF/lYI/Cxo=" crossorigin="anonymous"></script>	
 	</head>
 	<body>
-	
   		<button type ="submit"><a href="/comment">Comment</a></button>
   		<button type ="submit"><a href="/authorization">Sing in</a></button>
   		<button type ="submit"><a href="/registration">Create account</a></button>
-  	<form action="" method="POST">
-	  <input type="hidden" name="logout" value="true" />
-		<button type="submit">Sing out</button>
+  	<form action="/authorization/logout" method="POST">
+		<button name="logout_btn" type="submit" value="logout">Sing out</button>
 	</form>
 		<?php include 'application/views/'.$content_view; ?>
 	</body>
 </html>
 
-<?php
-if (isset($_POST['logout'])) {
-	unset($_SESSION['user_email']);
-	session_destroy();
+
+
+
+
+
+
+
+
+
```

Comment:
What is this for?

Reply:
I added css for button. removed these lines